docs/administration/upgrade/roll-back-upgrade.md

```diff
@@ -9,6 +9,13 @@ If, for any reason, you need to roll back to the previous version of Harbor, per
 To roll back from an upgrade, you must have backed up the previous version of Harbor. For information about backing up Harbor before an upgrade, see [Upgrade Harbor and Migrate Data](_index.md).
 {{< /note >}}
 
+{{< note >}}
+Because Harbor 2.2 is compiled by Golang 1.15, you must generate the internal certificates again, using command like this `docker run -v /:/hostfs goharbor/prepare:v2.2.0 gencert -p /path/to/internal/tls/cert
```

Comment:
Wording suggestion:

Harbor 2.2 is compiled by Golang 1.15 which deprecated self signed certificate without SAN. If you enabled internal TLS and cert files are generated by Harbor prepare script, you must generate the internal certificates again, using command like this `docker run -v /:/hostfs goharbor/prepare:v2.2.0 gencert -p /path/to/internal/tls/cert
`. The certs generated by the old version prepare script didn't include the SAN extension. If you manage the certs file, make the SAN is included, if not please check [Configure Internal TLS communication between Harbor Component](../../../install-config/configure-internal-tls). For more information about Go's break please refer [release note of Go 1.5](https://golang.org/doc/go1.15#commonname) and [this issue](https://github.com/golang/go/issues/24151).
{{< /note >}}

Reply:
thanks @a-mccarthy